BBallStatsTrack/src/com/example/bballstatstrack/models/Player.java

```diff
@@ -0,0 +1,265 @@
+    }
+
+    public int getBlock()
+    {
+        return mBlock;
+    }
+
+    public int getDefRebound()
+    {
+        return mDefRebound;
+    }
+
+    public int getFoulCount()
+    {
+        return mFoul;
+    }
+
+    public int getFTMade()
+    {
+        return mFTMade;
+    }
+
+    public int getFTMiss()
+    {
+        return mFTMiss;
+    }
+
+    public String getFullName()
+    {
+        return mFullName;
+    }
+
+    public int getNumber()
+    {
+        return mNumber;
+    }
+
+    public int getOffRebound()
+    {
+        return mOffRebound;
+    }
+
+    public int getPlayingTimeSec()
+    {
+        return mPlayingTimeSec;
+    }
+
+    public int getSteal()
+    {
+        return mSteal;
+    }
+
+    public int getTotalscore()
+    {
+        return getFTMade() + ( get2ptFGMade() * 2 ) + ( get3ptFGMade() * 3 );
+    }
+
+    public int getTurnover()
+    {
+        return mTurnover;
+    }
+
+    public void incrementPlayingTime()
+    {
+        mPlayingTimeSec++;
+    }
+
+    public void makeAssist()
+    {
+        mAssist++;
+    }
+
+    public void makeBlock()
+    {
+        mBlock++;
+    }
+
+    public void makeFoul()
+    {
+        mFoul++;
+    }
+
+    public void makeRebound( boolean offensive )
+    {
+        if( offensive )
+        {
+            mOffRebound++;
+        }
+        else
+        {
+            mDefRebound++;
+        }
+    }
+
+    public void makeSteal()
+    {
+        mSteal++;
+    }
+
+    public void makeTurnover()
+    {
+        mTurnover++;
+    }
+
+    public void setFullName( String fullName )
+    {
+        mFullName = fullName;
+    }
+
+    public void setNumber( int number )
+    {
+        mNumber = number;
+    }
+
+    public void shoot2pt( boolean made )
+    {
+        if( made )
+        {
+            m2ptFGMade++;
+        }
+        else
+        {
+            m2ptFGMiss++;
+        }
+    }
+
+    public void shoot3pt( boolean made )
+    {
+        if( made )
+        {
+            m3ptFGMade++;
+        }
+        else
+        {
+            m3ptFGMiss++;
+        }
+    }
+
+    public void shootFT( boolean made )
+    {
+        if( made )
+        {
+            mFTMade++;
+        }
+        else
+        {
+            mFTMiss++;
+        }
+    }
+
+    @Override
+    public String toString()
+    {
+        return getNumber() + " - " + getFullName();
+    }
+
+    public enum PlayerStats
+    {
+        NUMBER( "number" ), NAME( "fullName" ), MISS_1PT( "ftMiss" ), MADE_1PT( "ftMade" ), MISS_2PT(
+                "2ptFGMiss" ), MADE_2PT( "2ptFGMade" ), MISS_3PT( "3ptFGMiss" ), MADE_3PT(
+                        "3ptFGMade" ), OFFENSIVE_REBOUND( "offRebound" ), DEFENSIVE_REBOUND( "defRebound" ), ASSIST(
+                                "assist" ), TURNOVER( "turnover" ), STEAL( "steal" ), BLOCK( "block" ), FOUL(
+                                        "foul" ), PLAYING_TIME( "playingTimeSec" );
+
+        private final String mConstant;
```

Comment:
You can change this to public since its already final then use PlayerStats.mConstant po instead of overriding toString()